lib/tasks/active_stash.rake

```diff
@@ -174,3 +174,9 @@ namespace :active_stash do
     end
   end
 end
+
+Rake::Task["db:migrate"].enhance do
+  if ActiveStash::Assess.report_exists?
+    Rake::Task["active_stash:assess"].execute({quiet: true})
```

Comment:
We only want to run `active_stash:assess` after `db:migrate` if the assessment file already exists. The intent here is to make life easy for users that have already generated the report, but to not create the file if the user isn't using Assess.

We also suppress output to stdout to avoid extra noise during migrations.

Reply:
This is a smart way of handling this. 

Nice work @CDThomas.